torch_xla/csrc/xla_sharding_util.cpp

```diff
@@ -287,7 +287,8 @@ xla::OpSharding ShardingUtil::CreateOpSharding(
     }
   }
   TF_VLOG(INFO) << "OpSharding (ShardingType: " << sharding_type << "):\n"
-                << sharding.DebugString();
+                << sharding.DebugString()
+                << ", sharding.type()=" << sharding.type();
```

Comment:
DebugString should include the type?

Reply:
Actually, it doesn't. The debugString is empty in that case.